src/main/java/ome/formats/OMEROMetadataStoreClient.java

```diff
@@ -454,15 +468,18 @@ private void initializeServices(boolean manageLifecycle)
         //checkImmersions();
 
         // Start our keep alive executor
-        if (manageLifecycle)
+        if (keepAliveRate != null)
         {
-            if (executor == null)
-            {
-                executor = new ScheduledThreadPoolExecutor(1);
-                executor.scheduleWithFixedDelay(keepAlive, 60, 60, TimeUnit.SECONDS);
+            if (executor != null) {
+                log.error("Executor already set!");
+                executor.shutdown();
             }
+            keepAlive = new ClientKeepAlive();
+            keepAlive.setClient(this); // This is used elsewhere.
+            executor = new ScheduledThreadPoolExecutor(1);
+            executor.scheduleWithFixedDelay(keepAlive, keepAliveRate, keepAliveRate, TimeUnit.SECONDS);
```

Comment:
So I guess this isn't part of the normal OMERO server thread pool?

Reply:
Unfortunately not, which is either part of the problem or part of _another_ problem.

Reply:
see https://github.com/ome/omero-blitz/issues/77